egui_demo_lib/src/demo/context_menu.rs

```diff
@@ -116,17 +116,21 @@ impl super::View for ContextMenus {
 
 impl ContextMenus {
     fn example_plot(&self, ui: &mut egui::Ui) -> egui::Response {
-        use egui::plot::{Line, Value, Values};
+        use egui::plot::Line;
         let n = 128;
-        let line = Line::new(Values::from_values_iter((0..=n).map(|i| {
-            use std::f64::consts::TAU;
-            let x = egui::remap(i as f64, 0.0..=n as f64, -TAU..=TAU);
-            match self.plot {
-                Plot::Sin => Value::new(x, x.sin()),
-                Plot::Bell => Value::new(x, 10.0 * gaussian(x)),
-                Plot::Sigmoid => Value::new(x, sigmoid(x)),
-            }
-        })));
+        let line = Line::new(
+            (0..=n)
+                .map(|i| {
+                    use std::f64::consts::TAU;
+                    let x = egui::remap(i as f64, 0.0..=n as f64, -TAU..=TAU);
+                    match self.plot {
+                        Plot::Sin => [x, x.sin()],
+                        Plot::Bell => [x, 10.0 * gaussian(x)],
+                        Plot::Sigmoid => [x, sigmoid(x)],
+                    }
+                })
+                .collect::<Vec<_>>(),
```

Comment:
Is `.collect::<Vec<_>>()` really more ergonomic than `PlotPoints::from_iter(…)` ?

Reply:
Good idea, I implemented `FromIterator` for `PlotPoints` and used this in the examples.